dev/io.openliberty.pages.3.1.internal.factories/bnd.bnd

```diff
@@ -48,7 +48,7 @@ Import-Package: \
    jakarta.servlet;version="[6.0,7)", \
    jakarta.servlet.http;version="[6.0,7)", \
    jakarta.servlet.resources;version="[6.0,7)", \
-   jakarta.servlet.jsp;version="[3.1,4)" \
+   jakarta.servlet.jsp;version="[3.1,4)", \
```

Comment:
With the missing `,` didn't the rest of the imports declared after this get ignored?  Was the `Import-Package` really broken before here?

Reply:
Yes they were ignored.  I compared the manifest files before and after and they are equivalent due to bnd picking up the import ranges from the buildpath.

Reply:
Oh gosh....  Thank you for fixing it